src/Simplic.OxS.ModelDefinition.Extenstion.Abstractions/SourceAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Simplic.OxS.ModelDefinition.Extenstion.Abstractions
+{
+    [AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
+    public sealed class SourceAttribute : Attribute
+    {
+        private readonly string endpoint;
+        private readonly string sourceUrl;
+
+        public SourceAttribute(string sourceUrl, string endpoint)
+        {
+            this.endpoint = endpoint;
+            this.sourceUrl = sourceUrl;
+        }
+
+        public string SourceUrl
+        {
+            get => sourceUrl;
+        }
+
+        public string Endpoint
+        {
+            get => endpoint;
+        }
```

Comment:
I would suggest transform them in one-line statements, like:

```suggestion
        public string SourceUrl => sourceUrl;

        public string Endpoint => endpoint;
```